pupil/public/javascripts/mtc-check.js

```diff
@@ -1,25 +1,24 @@
-/* global Touch DocumentTouch Event */
-
 (function (global) {
   'use strict'
-  var GOVUK = global.GOVUK || {}
-  var startTime
-  var pageInterval = 5  // seconds allowed per page
-  var preloadInterval = 2 // seconds for student preparation
-  var deadline
 
-  var registerInputs = []
+  var GOVUK = global.GOVUK || {}
+  var defaultQuestionTimeLimit = 5  // seconds allowed per page
```

Comment:
I would have thought this would make maintenance harder as tracking down all these values requires prior knowledge.

If you don't share the same point of view then would you consider documenting it all somewhere?